web/src/services/memoService.ts

```diff
@@ -10,21 +11,26 @@ const convertResponseModelMemo = (memo: Memo): Memo => {
   };
 };
 
+const getUserIdFromPath = () => {
+  const path = locationService.getState().pathname.slice(1);
```

Comment:
I think we could update the route to `/u/:userId`. It's more clearly, and we could create a new route `/m/:memoId` for viewing a memo with id in the future.

Reply:
And use a regex `/^\/u\/(\d+)/` to match the `:userId`

Reply:
My meaning is about the frontend route not the backend API. The current looks like is `/:userId`, and the `/u/:userId` will be more suitable.

Reply:
sorry, i will correct it soon